build.override.targets

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0"  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <TraversalBuildDependsOn>
+      WcfSetup;
+      $(TraversalBuildDependsOn);
+      WcfCleanup;
+    </TraversalBuildDependsOn>
+  </PropertyGroup>
+  <!--if ServiceUri is set, the service has been already started on another machine-->
+  <Target Name="WcfSetup" Condition="($(WithCategories.Contains('OuterLoop')) or '$(OuterLoop)' == 'true') and ('$(ServiceUri)' == '') and $(OSEnvironment) == 'Windows_NT'" >
+    <Exec Command="$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\scripts\StartWCFSelfHostedSvc.cmd">
```

Comment:
You need to add ContinueOnError="true" to the Exec line for the manually started WCF service scenario to work.  Here is what happened when I started the WCF test service first and then tried to build:
```
c:\OSS\wcf>build /p:WithCategories=OuterLoop
Tools are already initialized.
[ 4:45:53.43] Building Native Libraries...
[ 4:45:54.91] Building Managed Libraries...
  Set Up test services
c:\OSS\wcf\build.override.targets(13,5): error MSB3073: The command "c:\OSS\wcf\\src\System.Private.ServiceModel\tools\scripts\StartWCFSelfHostedSvc.cmd" exited with code -1. [c:\OSS\wcf\build.proj]

c:\OSS\wcf\build.override.targets(13,5): error MSB3073: The command "c:\OSS\wcf\\src\System.Private.ServiceModel\tools\scripts\StartWCFSelfHostedSvc.cmd" exited with code -1. [c:\OSS\wcf\build.proj]

    0 Warning(s)
    1 Error(s)
Time Elapsed 00:00:04.13
[ 4:45:59.75] Build Exit Code = 1

c:\OSS\wcf>
```